### PR TITLE
[docs] Introduce guides in the news section

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -89,10 +89,8 @@ subprojects {
     detekt {
         debug = true
         toolVersion = usedDetektVersion
-        config = files(
-                project.rootDir.resolve("detekt-cli/src/main/resources/default-detekt-config.yml"),
```
Comment (schalkms) [inline build.gradle.kts]: Why is this resolve not necessary anymore?

File: build.gradle.kts [hunk 2]
```diff
@@ -272,32 +270,27 @@ dependencies {
...
+    parallel = true
+    disableDefaultRuleSets = true
+    buildUponDefaultConfig = true
     setSource(files(projectDir))
     include("**/*.kt")
     include("**/*.kts")
     exclude("resources/")
     exclude("build/")
-    config = files(
-            projectDir.resolve("detekt-cli/src/main/resources/default-detekt-config.yml"),
```
Comment (schalkms) [inline build.gradle.kts]: Here too

File: docs/_posts/2019-03-03-configure-detekt-on-root-project.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            inspectionsProfile = "$userHome/.idea/inspect.xml"
+            report = "project.projectDir/reports"
+            mask = "*.kt"
+        }
+    }
+}
+```
+
+There is a slight starting overhead for starting the _Kotlin_ compiler for each gradle module.
+So sometimes it makes sense to add an additional _detekt_ task which runs over the whole project and produce one big report.
```
Comment (schalkms) [inline docs/_posts/2019-03-03-configure-detekt-on-root-project.md]: produces

File: docs/_posts/2019-03-03-configure-detekt-on-root-project.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            report = "project.projectDir/reports"
+            mask = "*.kt"
+        }
+    }
+}
+```
+
+There is a slight starting overhead for starting the _Kotlin_ compiler for each gradle module.
+So sometimes it makes sense to add an additional _detekt_ task which runs over the whole project and produce one big report. 
+Such a setup could look like following in its simplest form:
```
Comment (schalkms) [inline docs/_posts/2019-03-03-configure-detekt-on-root-project.md]: the following

File: docs/_posts/2019-03-03-configure-detekt-on-root-project.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+repositories {
+    jcenter()
+}
+
+detekt {
+    input = files(rootProject.rootDir)
+    buildUponDefaultConfig = true
+}
+```
+Make sure to specify the `input` parameter or no sources get located and _detekt_ won't run!
```
Comment (schalkms) [inline docs/_posts/2019-03-03-configure-detekt-on-root-project.md]: `get located` sounds weird here

File: build.gradle.kts (hunk 1 above)
Comment (arturbosch) [reply build.gradle.kts]: WIth the new flag introduced with RC13, we do not need to specify the default config anymore.
See next line: `buildUponDefaultConfig = true`.
It helps not copy-pasting this file but rather focus on overwritting default behavior :)

**Comment by arturbosch** [comment]
Thanks for spell checking!

**Pull request merged**